### PR TITLE
added manifest for com.github.arshubham.gitignore

### PR DESCRIPTION
**Pull request by sahilarora3117**

### CODE REVIEW

**Comment by sahilarora3117** [comment]
bot, build com.github.arshubham.gitignore

File: com.github.arshubham.gitignore.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.9.tar.xz",
+					"sha256": "699d76a453e6a3d3331906346e3dbfa25f2cbc9ec090e46635e9c6bb595e07c2"
+				}
+			]
+		},
+        {
+            "name": "Cipher",
```
Comment (bilelmoussaoui) [inline com.github.arshubham.gitignore.json:29]:
```suggestion
            "name": "gitignore",
```

File: com.github.arshubham.gitignore.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "base-version" : "juno-20.08",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.arshubham.gitignore",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
```
Comment (bilelmoussaoui) [inline com.github.arshubham.gitignore.json:15]: What does it need home access for?

Comment (sahilarora3117) [reply com.github.arshubham.gitignore.json:15]: That's for saving the gitignore files.

Comment (bilelmoussaoui) [reply com.github.arshubham.gitignore.json:15]: A NativeFileChooser should work just fine without the explicit home permission.

Comment (sahilarora3117) [reply com.github.arshubham.gitignore.json:15]: I tried doing that, but it doesn't allow the saving of files into the system

**Comment by barthalion** [comment]
bot, build com.github.arshubham.gitignore

**Pull request closed without merging**